simSPI/cryoemio.py

```diff
@@ -0,0 +1,314 @@
+"""Helper functions for tem.py processing of input and output files."""
```

Comment:
On a second read, this should probably be in ioSPI: what do you think?

Reply:
my understanding is that ioSPI is reserved for processes related to data flowing between user and OSF cloud - maybe might be best to get more input but I'd prefer to leave it in simSPI for the time being unless others think different.

Reply:
(also tests are coming soon for cryoemio.py)

Reply:
The I/O in ioSPI refers to https://en.wikipedia.org/wiki/Input/output
and the purpose of that repository is to have all necessary functions and methods to read and write any type of file that contains data, results, etc.
In that sense, I think the functions here belong in ioSPI.